src/sentry/integrations/pagerduty/client.py

```diff
@@ -27,16 +28,17 @@ def send_trigger(self, data):
         if isinstance(data, EventCommon):
             source = data.transaction or data.culprit or "<unknown>"
             group = data.group
+            custom_details = serialize(data, None, ExternalEventSerializer())
             payload = {
                 "routing_key": self.integration_key,
                 "event_action": "trigger",
                 "dedup_key": group.qualified_short_id,
                 "payload": {
                     "summary": data.message or data.title,
-                    "severity": "error",
+                    "severity": data.get_tag("level") or "error",
```

Comment:
What are the values pagerduty expects for "severity"? Surprised to see them line up with our concept of "level"?

Reply:
Yeah, we should definitely have an explicit mapping